nav2_behavior_tree/CMakeLists.txt

```diff
@@ -263,6 +280,11 @@ install(DIRECTORY test/utils/
   DESTINATION include/${PROJECT_NAME}/nav2_behavior_tree/test/utils
 )
 
+install(DIRECTORY cmake
+  DESTINATION share/${PROJECT_NAME}
+)
+
+#NOTE: Still using manually managed nav2_tree_nodes.xml in source directory
```

Comment:
Should we though? If we removed this, what would be the workflow for using groot?